### ISSUE TITLE
Add accessibility props on images and touchable components

### ISSUE THREAD

**Issue by JonEsparaz** [jesus-collective/mobile#222]
We should be adding accessibility properties to a ton of components (basically everywhere). My suggestion is we add the accessibility props in one shot after V2 is complete, but before the public launch in the fall. 

@lucastbelem let us know if you think we need to expedite the timeline on this.

**Comment by lucastbelem** [comment]
@GeorgeBellTMH how much of this is still needed?

**Comment by lucastbelem** [comment]
@itbel here is one.
